GameData/RealismOverhaul/RO_SuggestedMods/RO_B9.cfg

```diff
@@ -675,7 +675,7 @@
 	{
 		@name = AJEInlet
 		Area = 1.2
-		#@AJE_TPR_CURVE_DEFAULTS/FixedCone/TPRCurve {}
+		//@AJE_TPR_CURVE_DEFAULTS/FixedCone/TPRCurve {}
```

Comment:
I think this was maybe trying to copy a node rather than be a comment. @DRVeyl or @al2me6 will know the syntax for sure.

Reply:
Thanks, I have just now learned about the "paste" operator. :+1: